app/controllers/application_controller.rb

```diff
@@ -232,17 +230,9 @@ def require_training!
 
   def check_all_db_migrations
     return true unless Rails.env.development?
-
-    GrdaWarehouseBase.setup_config
-    raise ActiveRecord::MigrationError, "Warehouse Migrations pending. To resolve this issue, run:\n\n\t bin/rake warehouse:db:migrate RAILS_ENV=#{::Rails.env}" if GrdaWarehouseBase.needs_migration?
-
-    HealthBase.setup_config
-    raise ActiveRecord::MigrationError, "Health Migrations pending. To resolve this issue, run:\n\n\t bin/rake health:db:migrate RAILS_ENV=#{::Rails.env}" if HealthBase.needs_migration?
-
-    ReportingBase.setup_config
-    raise ActiveRecord::MigrationError, "Reporting Migrations pending. To resolve this issue, run:\n\n\t bin/rake reporting:db:migrate RAILS_ENV=#{::Rails.env}" if ReportingBase.needs_migration?
-
-    ApplicationRecord.setup_config
+    raise ActiveRecord::MigrationError, "Warehouse Migrations pending. To resolve this issue, run:\n\n\t bin/rails db:migrate:warehouse RAILS_ENV=#{::Rails.env}" if GrdaWarehouseBase.needs_migration?
```

Comment:
Do we need this in rails 6 or is it part of the core logic now?  (Fine to leave it for now if it doesn't break anything)

Reply:
I'm not sure.